### PR TITLE
Simplify navigation on variables page

### PR DESCRIPTION
**Pull request by bear454**
Save data and move on to plan by default. If there is a problem, fall back to variables form.

This was the behavior of the 'plan' button anyway; just dropping the redundant 'save' button.

### CODE REVIEW

**Comment by bear454** [comment]
Screenshot_2020-10-23 blue-horizon demo

File: app/controllers/variables_controller.rb [hunk 1]
```diff
@@ -12,18 +12,9 @@ def show
...
     if @variables.save
-      flash_message = {}
-      if params[:button]
-        target_path = plan_path
-      else
-        flash_message = { notice: 'Variables were successfully updated.' }
-        target_path = variables_path
-      end
-      redirect_to target_path, flash: flash_message
+      redirect_to plan_path
```
Comment (MalloZup) [inline app/controllers/variables_controller.rb:17]: I think you could avoid the `if/else` logic and use just IF as guard clause.

```ruby
  def update
   ... 

     redirect_to plan_path if @variables.save 
     redirect_to variables_path, flash: {
        error: @variables.errors.full_messages
      }
      
```

Comment (bear454) [reply app/controllers/variables_controller.rb:17]: Nice catch, thanks.

**Review by arbulu89 — APPROVED** [review]
Looks good!

**Review by jesusbv — APPROVED** [review]

**Pull request merged**